src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -228,13 +228,13 @@ public partial class AuthenticationResult
         /// </summary>
         public Guid CorrelationId { get; }
 
-        internal ApiEvent ApiEvent { get; }
-
         /// <summary>
         /// Identifies the type of access token. By default tokens returned by Azure Active Directory are Bearer tokens.        
         /// <seealso cref="CreateAuthorizationHeader"/> for getting an HTTP authorization header from an AuthenticationResult.
         /// </summary>
-        internal string TokenType { get; }
+        public string TokenType { get; }
```

Comment:
Since you made this property public, you must also update the ctor of AuthenticationResult, otherwise developers will not be able to test properly. We had a customer issue with this in the past, so I even wrote a test `PublicTestConstructorCoversAllProperties` (which is happily failing on this PR).

Reply:
thx @bgavrilMS the comment really threw me off. LOL